Automattic/wp-e2e-tests
# Test import in signup flow

Adds a test for the following import signup flow to `wp-signup-spec`:

1. Visit `/start/import`
1. Enter various invalid urls and expect an error
1. Enter valid url and continue
1. Retry if site importability check fails the first time
1. Enter account details
1. Confirm account email
1. Land on importers page and see preview of site to be imported

## To test locally

1. Set `calypsoBaseURL` to `https://wpcalypso.wordpress.com`
1. Run `wp-signup-spec` `env NODE_ENV=my-config ./node_modules/.bin/mocha specs/wp-signup-spec.js`

To test retry when site importability check fails, sandbox public-api.wordpress.com and remove/modify `is-site-importable` endpoint.
